frontend/lib/tracking/ga.ts

```diff
@@ -0,0 +1,78 @@
+interface TrackerConfig {
+    name: string;
+    id: string;
+    sampleRate: number;
+}
+interface Config {
+    tracker: TrackerConfig;
+    pillar: Pillar;
+}
+
+// const getAnalyticsEdition = () => {
+//     const edition = (guardian.config.page.edition || '').toLowerCase();
+//     return edition === 'int' ? 'international' : edition;
+// };
+
+export const init: (_: Config) => void = ({ tracker, pillar }) => {
+    const coldQueue = (...args: any[]) => {
+        (ga.q = ga.q || []).push(args);
+    };
+
+    ga = ga || (coldQueue as UniversalAnalytics.ga);
+    ga.l = +new Date();
+    ga('create', tracker.id, 'auto', '@tracker.trackerName', {
+        sampleRate: 100, // WARNING: we are sampling everyone!
+        siteSpeedSampleRate: tracker.sampleRate,
+        // TODO: 'userId': identityId
+    });
```

Comment:
You probably do not want to introduce this at this time, but have you read and consider [idle until urgent](https://philipwalton.com/articles/idle-until-urgent/) strategy?
```js
const queue = new IdleQueue({ensureTasksRun: true});
queue.pushTask(() => ga(
                 'create',
                  tracker.id,
                 'auto',
                 '@tracker.trackerName', {
                                sampleRate: 100, // WARNING: we are sampling everyone!
                                siteSpeedSampleRate: tracker.sampleRate,
                                // TODO: 'userId': identityId
                 });
              );
```